packages/validation/src/parseValues/parseValues.ts

```diff
@@ -138,7 +138,7 @@ const convertArrayValue = (value: string, fieldDefinition: SchemaField): Result<
 
 	/* === Start of convertArrayValue logic === */
 	const { valueType } = fieldDefinition;
-	const delimiter = DEFAULT_DELIMITER;
+	const delimiter = fieldDefinition.delimiter !== undefined ? fieldDefinition.delimiter : DEFAULT_DELIMITER;
```

Comment:
I think this would allow empty strings as a delimiter, which in effect is not a delimiter at all. am I missing something here?
possible fix/workaround:
```suggestion
	const delimiter = typeof fieldDefinition.delimiter === 'string' && fieldDefinition.delimiter.length ? fieldDefinition.delimiter : DEFAULT_DELIMITER;
```

Reply:
Just realised zod schema states min 1... code here could still benefit from the additional declarativeness, though